From e66a0480afc4de7ca3863899df66b6d3e738f3de Mon Sep 17 00:00:00 2001
From: Joel Williams <joel.williams@codethink.co.uk>
Date: Mon, 20 Jun 2022 10:25:14 +0100
Subject: [PATCH 2/2] Include dt-bindings/pwm/pwm.h in hifive-unmatched-a00.dts

Added line in included to ensure dt-bindings/pwm/pwm.h is included in file.
---
 arch/riscv/boot/dts/sifive/hifive-unmatched-a00.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/riscv/boot/dts/sifive/hifive-unmatched-a00.dts b/arch/riscv/boot/dts/sifive/hifive-unmatched-a00.dts
index 181e0b53ca55..7f384f2521e7 100644
--- a/arch/riscv/boot/dts/sifive/hifive-unmatched-a00.dts
+++ b/arch/riscv/boot/dts/sifive/hifive-unmatched-a00.dts
@@ -3,6 +3,7 @@
 
 #include "fu740-c000.dtsi"
 #include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/pwm/pwm.h>
 #include <dt-bindings/leds/common.h>
 #include <dt-bindings/interrupt-controller/irq.h>
 
-- 
2.30.2

